src/ExpEvaluator.cpp

```diff
@@ -9,7 +9,7 @@
 ExpEvaluator::ExpEvaluator() {
   _interpolate = true;
   _exp_table = NULL;
-  _polar_quad = NULL;
+  _quad = NULL;
```

Comment:
Ditto, inconsistent naming convention for the `Quadrature` class attribute between `Solver`, `Cmfd` and `ExpEvaluator`